### PR TITLE
CIV-7083 Dependabot replaced by Renovate

### PR DESCRIPTION
**Pull request by UshaPanneerselvam1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-7083

### Change description ###
Dependabot is replaced by Renovate

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by filippo-possenti-version1 — APPROVED** [review]

**Comment by vasudevganesanhmcts** [comment]
@UshaPanneerselvam1  @filippo-possenti-version1  please dont merge this as i need to fix the pipelines . .i will merge this once its stable thanks

**Review by vasudevganesanhmcts — APPROVED** [review]

**Pull request merged**
